core/src/main/java/org/geysermc/geyser/translator/inventory/item/ItemTranslator.java

```diff
@@ -116,6 +115,11 @@ private static ItemData.Builder translateToBedrock(GeyserSession session, Item j
         }
 
         nbt = translateDisplayProperties(session, nbt, bedrockItem);
+
+        if (nbt != null && nbt.contains("AttributeModifiers")) {
```

Comment:
You already check for null in addAttributes; you don't need this `contains` check.

Reply:
Thanks, fixed